specflow/MVP-Advance/SeleniumSpecFlow/Features-Definitions/SkillSwap/ShareSkillStepDefinitions.cs

```diff
@@ -22,9 +23,8 @@ public void WhenIEnterMySkillDetails()
         {
             shareSkillObj.EnterShareSkill(2, "ManageListings");
         }
-
-        [Then(@"I view my skill details based on title")]
-        public void ThenIViewMySkillDetailsBasedOnTitle()
+        [When(@"I view my skill details based on title")]
+        public void WhenIViewMySkillDetailsBasedOnTitle()
         {
             shareSkillObj.ViewMySkillDetails(2, "ManageListings");
```

Comment:
the ShareSkill shareSkillObj;  can be moved inside a constructor, not a must but  good practice!